packages/app/src/containers/Canvas.tsx

```diff
@@ -433,6 +435,25 @@ export default memo(function Canvas() {
           event.preventDefault();
           break;
         }
+        case 'maybeConvertCurveMode': {
+          const { origin } = state.interactionState;
+
+          if (isMoving(point, origin)) {
+            dispatch('setPointCurveMode', Sketch.CurveMode.Mirrored);
+            dispatch(
+              'selectControlPoint',
+              selectedLayers[0].do_objectID,
+              0,
+              'curveFrom',
+            );
+            dispatch('interaction', ['maybeMoveControlPoint', origin]);
```

Comment:
This `maybe` state isn't really necessary here, so I'll push a quick fix to remove it after we merge this PR. I might have to change things in a couple other places though, so I'll do it separately.